k8s/aat/common/adoption/adoption-backend.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: adoption-backend
+  namespace: adoption
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: adoption-backend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: adoption-backend
+    version: 1.3.0
```

Comment:
This should match https://github.com/hmcts/adoption-backend/blob/master/charts/adoption-backend/Chart.yaml#L6